pkg/providers/common/auditpolicy.go

```diff
@@ -0,0 +1,252 @@
+package common
+
+import (
+	auditv1 "k8s.io/apiserver/pkg/apis/audit/v1"
+	"sigs.k8s.io/yaml"
+
+	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+)
+
+func GetAuditPolicyYaml(kubeVersion anywherev1.KubernetesVersion) ([]byte, error) {
+	return yaml.Marshal(GetAuditPolicy(kubeVersion))
+}
+
+func GetAuditPolicy(kubeVersion anywherev1.KubernetesVersion) *auditv1.Policy {
+	a := baseAuditPolicy()
+	if kubeVersion == anywherev1.Kube124 {
+		a.APIVersion = "audit.k8s.io/v1"
+	} else {
+		a.APIVersion = "audit.k8s.io/v1beta1"
+	}
```

Comment:
if we'll be using v1 as the version going forward it seems like we should default to v1 instead of defaulting to v1beta1
```suggestion
	if kubeVersion in [anywherev1.Kube120, anywherev1.Kube121, anywherev1.Kube122, ..., anywherev1.Kube123] {
		a.APIVersion = "audit.k8s.io/v1beta1"
	} else {
		a.APIVersion = "audit.k8s.io/v1"
	}
```